### PR TITLE
Add Download tab to SCEA

### PR DESCRIPTION
**Pull request by haideriqbal**
Added download tab to SCEA similar to bulk expression atlas.

### CODE REVIEW

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <p>
+            <img src="/gxa/sc/resources/images/download/download-tsv-file.png" alt="idf and sdrf file" align="bottom" class="icon"/>
+            Experiment Metadata: SDRF and IDF files containing information about the experiment.
+        </p>
+        <p>
+            <img src="/gxa/sc/resources/images/download/download-experiment-design.png" alt="experiment design file" align="bottom" class="icon"/>
+            Experiment Design file: Information about assays within the experiment.
+        </p>
+
+        <h4>Results Files</h4>
```
Comment (lingyun1010) [inline app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp:30]: We also added some new files, summary pdf https://www.pivotaltracker.com/story/show/179434417 and Unprocessed protein groups file  https://www.pivotaltracker.com/story/show/181094306. We also import some new icons with <i> tag or images in the atlas-experiments component. Please double check the file icons.

Comment (haideriqbal) [reply app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp:30]: @lingyun1010 the pivotal stories you mentioned refer to the new files added to the bulk expression atlas. I do not see any new types of files added to the download section for the single-cell experiments to which this PR is referring. Can you please check and let me know if any files are added for single cell experiments?

Comment (lingyun1010) [reply app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp:30]: Yes, you are right. No new files added to SC as far as I know.

**Comment by lingyun1010** [comment]
LTGM!

**Review by alfonsomunozpomer — COMMENTED** [review]
May I ask where the two PNGs come from? They look terribly compressed.

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+        <h3>From experiment pages</h3>
+        <p>
+            You can download single experiment files from the <i>Downloads</i> tab in any experiment page. Below is
+            a short summary about data that you can download.
+        </p>
+
+        <h4>Metadata Files</h4>
+        <p>
+            <img src="/gxa/sc/resources/images/download/download-tsv-file.png" alt="idf and sdrf file" align="bottom" class="icon"/>
```
Comment (alfonsomunozpomer) [inline app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp]: Please properly capitalise text in `alt` field.

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+<div class="row">
+    <div class="12 columns">
+        <h2>Download</h2>
+
+        <h3>Via FTP</h3>
+        <p>
+            You can download data for every dataset individually in Expression Atlas through our
+            <a href="http://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/experiments/" target="_blank">FTP site</a>.
```
Comment (alfonsomunozpomer) [inline app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp]: This is linking to bulk experiments, please point at the right URL.

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        </p>
+
+        <h3>From experiment pages</h3>
+        <p>
+            You can download single experiment files from the <i>Downloads</i> tab in any experiment page. Below is
+            a short summary about data that you can download.
+        </p>
+
+        <h4>Metadata Files</h4>
```
Comment (alfonsomunozpomer) [inline app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp]: Files should be in lower case.

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        <p>
+            <img src="/gxa/sc/resources/images/download/download-tsv-file.png" alt="idf and sdrf file" align="bottom" class="icon"/>
+            Experiment Metadata: SDRF and IDF files containing information about the experiment.
```
Comment (alfonsomunozpomer) [inline app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp]: Metadata should be in lowercase.

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        </p>
+        <p>
+            <img src="/gxa/sc/resources/images/download/download-experiment-design.png" alt="experiment design file" align="bottom" class="icon"/>
```
Comment (alfonsomunozpomer) [inline app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp]: Capitalise `alt` field.

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+            Experiment Design file: Information about assays within the experiment.
```
Comment (alfonsomunozpomer) [inline app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp]: Design should be in lower case.

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
I left some detailed comments on things that should be changed. Please have a look.

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp (hunk 3 above)
Comment (haideriqbal) [reply app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp]: Doesn't this url contains both single cell and bulk experiments? For example, I can see [E-MTAB-5061](http://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/experiments/E-MTAB-5061/) there which is a single cell experiment.

**Comment by haideriqbal** [comment]
> May I ask where the two PNGs come from? They look terribly compressed.
Screenshot 2022-03-28 at 18 53 58

I got these from experiment-page bundle in single cell.

File: app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp (hunk 3 above)
Comment (alfonsomunozpomer) [reply app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp]: Could you please raise this issue with curators and data production? I could be wrong and we want both sites to point at a centralised directory with both bulk and SC experiments.

Comment (haideriqbal) [reply app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp]: sure will ask curators/data prod team!

**Pull request merged**